examples/super_resolution/main.py

```diff
@@ -45,7 +47,7 @@
 
 
 class SRDataset(torch.utils.data.Dataset):
-    def __init__(self, dataset, scale_factor, crop_size=256):
+    def __init__(self, dataset, scale_factor, crop_size=180):
```

Comment:
If you tweak this value for CI, let's make it as a parameter and keep default as 256

Reply:
ok let me do that

Reply:
Hey @vfdev-5 , Actually I tested it again and it may not work for `crop_size=256` always as some images may have a lower size than 200 pixels. So I have made it an argument of `main.py` that can be variable depending upon the choice of dataset for training and the `upscale_factor`